Classes/Utility/PathUtility.php

```diff
@@ -19,36 +18,21 @@
 class PathUtility {
 
 	/**
-	 * @var array
-	 */
-	private static $knownPathNames = array(
```

Comment:
We can drop the legacy names but we do need to check for the path being one of the recognised paths - the reason is that the view configuration array may contain other flags (`enabled` which is used by Fluidpages, for example) and we don't want to translate those.